IntegrationAdapters/Services/MedicineReportService.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace IntegrationAdapters.Services
 {
-    public class MedicineReportService
+    public class MedicineReportService : IMedicineReportRepository

```

Comment:
Why does service implement repository interface?

Reply:
Ups, lapsus.. 
It was supposed to be a MedicineReportRepository :(

Reply:
It's ok hahaha you know what to fix then 😄